src/schemas/json/uplugin.json

```diff
@@ -0,0 +1,401 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "additionalProperties": false,
+  "description": "Descriptor for plugins. Contains all the information contained within a `.uplugin` file.",
+  "definitions": {
+    "BuildConfiguration": {
+      "description": "Available build configurations. Mirorred from `UnrealTargetConfiguration`.",
+      "type": "string",
+      "enum": [
+        "Unknown",
+        "Debug",
+        "DebugGame",
+        "Development",
+        "Shipping",
+        "Test"
+      ]
+    },
+    "BuildTargetType": {
+      "description": "Enumerates build target types.",
+      "type": "string",
+      "enum": ["Unknown", "Game", "Server", "Client", "Editor", "Program"]
+    },
+    "ModuleDescriptor": {
+      "description": "Description of a loadable module.",
+      "type": "object",
+      "properties": {
+        "AdditionalDependencies": {
+          "description": "List of additional dependencies for building this module.",
+          "type": "array",
```

Comment:
Optional fix: add `"uniqueItems": true` to make sure that there are no duplicates.

Reply:
Optional fix implemented in 262cb20dcfc106d02304efb65aa070fa0db880cc.